src/components/ModalDelete/ModalDeleteComponent.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Slide from '@material-ui/core/Slide';
+import { useDispatch, useSelector } from 'react-redux';
+import { deleteNoteAction } from 'components/BottomBar/BottomBarActions';
+import { makeStyles } from '@material-ui/core';
+import { colors } from '../../colors';
+
+const useStyles = makeStyles(() => ({
+  dialog: {
+    color: 'white',
+    backgroundColor: colors.mainGray,
```

Comment:
Shouldn't we use `colors.dialogsGray` instead of `colors.mainGray` here?

![image](https://user-images.githubusercontent.com/6686679/115607531-c3bfae00-a2ed-11eb-992a-1c077c813c18.png)